_posts/2019-04-16-sicko-mode.md

```diff
@@ -0,0 +1,100 @@
+they note our character is “goin’ on you with the pick and roll”. For readers
+unfamiliar with basketball, a pick and roll involves a player forcing a
+defender to *switch* to defending him by setting up a screen. These words are
+accompanied by a beat switch -- and also accompanied by our character’s switch
+of sexuality.
+
+The second beat is more aggressive and powerful, consistent with our
+character’s newfound interest in masculinity.
+
+> Different coloured chains, think my jeweller really sellin’ fruits
+
+ He immediately affirms his new sexuality by buying multicoloured jewellery,
+which is associated with gay pride. Furthermore “fruit” is common slang for
+homosexuals, further alluding to his jeweller selling symbols of homosexuality.
+
+> Most of these girls ain’t got a clue
+
+Not only does our character no longer see women as potential sexual partners,
+he goes so far as to deride their intelligence as a whole.
+
+> We ain’t even make it to room.  
+> She thought it was the ocean, it’s just a pool.
+
+Here, as a woman attempts to engage in promiscuous behaviour, our hero denies
+her access to his room (and therefore himself). Furthermore, he corrects her --
+his desire for women cannot be compared to the vast ocean, but just a small
+pool, by comparison practically non-existent.
+
+However, due to the anti-homosexual sentiments still present in many American
+communities, our protagonist is unable to sustain his true desires. Once they
+find out, he is physically assaulted and knocked “out like a light **(ey-uh)**”
+(emphasis mine). The beat change here again represents another change in his
+life. He must settle for practising his sexuality in secret and pretending to
+obey heterosexual norms.
+
+He regrets this as he notes “this shit got double bedroom”, meaning he
+unfortunately has to physically separate each of his sexualities - one bedroom
+for the women in his life, and another for the men. He must publicly proclaim
+“checks over stripes” (stripes on a multi coloured flag represent
+homosexuality). Despite his suffering, our strong and resilient protagonist
+still proclaims:
+
+> See the shots that I took (Ayy), wet like I'm Book (Ayy)  
+> Wet like I'm Lizzie
+
+or that he is able to get wet (i.e. ejaculate) both with Book (presumably a
+man) and Lizzie (presumably a woman). He is able to lead his double life well.
+
+Unfortunately, as the song title suggests, he is eventually unable to
+conclusively assert his preferences. He remains in fear of being branded a
+“sicko”, and this song is a sad reminder of that time his peers had thought he
+entered “sicko mode” (i.e homosexuality). Travis Scott’s wakeup call for
+America (and more generally, the world at large) to accept what is a natural
+phenomenon is brave, commendable, and well crafted.
+
+For those further interested, my colleagues DJ Splash Drip and Kurosare have
+taken this analysis further in their song “[Dicko
+Mode](https://www.youtube.com/watch?v=ycd6Xchkq-4)”.
```

Comment:
either swap period and quote or remove quotes altogether

Reply:
It does not make sense to have to the period inside. It is not part of the song title.

Reply:
Sire, I refer you to https://www.grammarly.com/blog/quotation-marks/

![image](https://user-images.githubusercontent.com/5828246/56323277-2dfe5080-6120-11e9-9ba6-f18c62eeeb87.png)

Reply:
Sire, I refer _you_ to the Economist and Guardian style guide.

https://english.stackexchange.com/questions/23/is-it-ever-acceptable-for-a-period-to-come-after-a-quote-at-the-end-of-a-sentenc

Reply:
Gentlemen, you both are correct. In American English, periods and commas go inside the quotation mark with no exceptions. However, in British English, the same thing applies for quotes being used for dialogue, but when used to refer to titles or specific terms, the quoted portion is treated as an atomic unit, with punctuation going outside.
Personally, I write in the style of the latter, and favor the British style, as the American style is a anarchic convention. Please refer to https://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks:

> Compositors―people who layout printed material with type―made the original rule that placed periods and commas inside quotation marks to protect the small metal pieces of type from breaking off the end of the sentence. The quotation marks protected the commas and periods. In the early 1900s, it appears that the Fowler brothers (who wrote a famous British style guide called The King’s English) began lobbying to make the rules more about logic and less about the mechanics of typesetting. They won the British battle, but Americans didn’t adopt the change.

Reply:
Sires, this is an **American** publication, produced by **Americans**. As such, we will conform to the native language, and accommodate our audience.

Reply:
**An open letter:**
I agree that we are Americans. However, what is America but a melting-pot of cultures, where the best aspects of societies around the globe are pieced together so that any one who graces this land has an opportunity for success? Truly, the **American** thing to do here is not to blindly follow convention set by an older generation, but rather, to innovate and improve upon their wisdom. Much like the theory of the living Constitution, I believe that grammar is a set of standards that should constantly adapt to the changing circumstances and constraints that a people face, not the other way around. As such, I am in favor of the situational placement of commas/periods around quotation marks.

Reply:
I offer you two pieces of evidence:

1. An [article](https://www.nytimes.com/2019/04/17/opinion/tsa-transgender.html) from The New York Times. You will note that all quotations contain punctuation inside.
2. An [article](https://www.theguardian.com/politics/2019/apr/17/theresa-may-could-put-off-queens-speech-amid-brexit-turmoil) from The Guardian, a British publication. You will note that all quotations contain punctuation inside.

Reply:
I must clarify, though, that while I am a proponent of British grammar rules for their flexibility and utilitarian focus, spelling should be done in an American manner, as our audience is primarily American, and there is nothing intrinsic about the way a word is spelled, as opposed to how words are laid out grammatically.

Reply:
If it is the goal of (dingers)[dingers-music.github.io] to enter the mainstream and entertain the general populace, we must apply proper written language rules.

Reply:
> in order to “build the broadest possible consensus for our Brexit plans”.
> refusal to bring forward a Queen’s speech would be a “constitutional outrage”.
The two excerpts above were taken from The Guardian's article. To briefly summarize, when dialogue is molded to fit the context of a sentence, sometimes referred to as blending quotations, as in the examples above, the quoted portion is treated as an atomic unit, which explains why the punctuation is on the outside.

Reply:
I concede

Reply:
But I still bear a preference for American. You may create your posts with the disclaimer that your writing employs archaic British effects; I will continue to create my publications the *correct* way.

Reply:
The polarization of today's grammar circles is quite disheartening. We must cast aside our grammatical preferences and realize that both parties just want what's best for our country semantically.

Reply:
Allow us to subscribe to the [AP Style book](https://www.apstylebook.com/) convention to eliminate any debate

Reply:
Sires, I believe us all to be logical men of the 21st century.

Let us cast aside all superstition and archaic remnants of the past!